frontend/src/lib/components/AuthorizedUrlList/authorizedUrlListLogic.ts

```diff
@@ -190,19 +190,23 @@ export interface AuthorizedUrlListLogicProps {
     actionId: number | null
     experimentId: ExperimentIdType | null
     type: AuthorizedUrlListType
-    query: string | null | undefined
     allowWildCards?: boolean
 }
 
 export const defaultAuthorizedUrlProperties = {
     actionId: null,
     experimentId: null,
-    query: null,
 }
 
 export const authorizedUrlListLogic = kea<authorizedUrlListLogicType>([
     path((key) => ['lib', 'components', 'AuthorizedUrlList', 'authorizedUrlListLogic', key]),
-    key((props) => (props.experimentId ? `${props.type}-${props.experimentId}` : `${props.type}-${props.actionId}`)),
+    key((props) =>
+        props.experimentId
```

Comment:
i know you didn't add this but we could just template all these things into the string regardless of presence... it doesn't matter if we ahve `undefined` in the key so long as its stable and it is one less thing to think about

total nitpick though

Reply:
That's an excellent point, thanks guys :)